cmd/admin-handlers-pools.go

```diff
@@ -391,3 +376,24 @@ func (a adminAPIHandlers) RebalanceStop(w http.ResponseWriter, r *http.Request)
 	writeSuccessResponseHeadersOnly(w)
 	logger.LogIf(ctx, pools.saveRebalanceStats(GlobalContext, 0, rebalSaveStoppedAt))
 }
+
+func proxyDecommissionRequest(ctx context.Context, defaultEndPoint Endpoint, w http.ResponseWriter, r *http.Request) (proxy bool) {
+	ep := os.Getenv("_MINIO_DECOM_ENDPOINT")
```

Comment:
This is temporary for now - we will add a more official one later.
```suggestion
	host := env.Get("_MINIO_DECOM_ENDPOINT_HOST", "")
```